x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/risk_score/all/index.ts

```diff
@@ -9,36 +9,36 @@ import type { IEsSearchResponse } from '@kbn/data-plugin/common';
 import type { SecuritySolutionFactory } from '../../types';
 import type {
   RiskScoreRequestOptions,
-  RiskScoreStrategyResponse,
   RiskQueries,
 } from '../../../../../../common/search_strategy';
 import { inspectStringifyObject } from '../../../../../utils/build_query';
 import { buildRiskScoreQuery } from './query.risk_score.dsl';
 import { DEFAULT_MAX_TABLE_QUERY_SIZE } from '../../../../../../common/constants';
 import { getTotalCount } from '../../cti/event_enrichment/helpers';
 
-export const riskScore: SecuritySolutionFactory<RiskQueries.riskScore> = {
+export const riskScore: SecuritySolutionFactory<
+  RiskQueries.hostsRiskScore | RiskQueries.usersRiskScore
+> = {
   buildDsl: (options: RiskScoreRequestOptions) => {
     if (options.pagination && options.pagination.querySize >= DEFAULT_MAX_TABLE_QUERY_SIZE) {
       throw new Error(`No query size above ${DEFAULT_MAX_TABLE_QUERY_SIZE}`);
     }
 
     return buildRiskScoreQuery(options);
   },
-  parse: async (
-    options: RiskScoreRequestOptions,
-    response: IEsSearchResponse<unknown>
-  ): Promise<RiskScoreStrategyResponse> => {
+  parse: async (options: RiskScoreRequestOptions, response: IEsSearchResponse) => {
     const inspect = {
       dsl: [inspectStringifyObject(buildRiskScoreQuery(options))],
     };
 
     const totalCount = getTotalCount(response.rawResponse.hits.total);
-
+    const hits = response?.rawResponse?.hits?.hits;
+    const data = hits?.map((hit) => hit._source) ?? [];
```

Comment:
I feel like we've been trying to migrate to primarily using the `fields` api, but not sure if the UI you have set up is set for that?

Reply:
I think the UI hasn't been updated for that, I'll create another PR to implement this.

Reply:
Thanks for pointing this out Michael!